src/timeline.js

```diff
@@ -107,7 +108,7 @@ export default class Timeline extends React.Component {
     forceRedrawFunc: null,
     onItemHover() {},
     onItemLeave() {},
-    interactOptions: {},
+    interactOptions: {}
```

Comment:
I see a lot of format modifications, such as this. It pollutes the diff; it's better to have a heavy reformatting commited separately.